pkg/controllers/mcsendpointslice/endpointslicesync_controller.go

```diff
@@ -0,0 +1,269 @@
+/*
```

Comment:
cc @RainbowMango @XiShanYongYe-Chang @chaunceyjiang 
Please give some suggestion about the directory name

Reply:
- svcnative ?